apps/server/src/modules/task/entity/task-with-status.vo.ts

```diff
@@ -0,0 +1,18 @@
+import { Task } from './task.entity';
+
+interface ITaskStatus {
+	submitted: number;
+	maxSubmissions: number;
+	graded: number;
+}
+
+export class TaskWithStatusVo {
```

Comment:
shouldn't that be part of the tasks entity?

Reply:
well, i also think that the task entity should provide it's status by itself. we should change that. i didn't want to cause too much trouble at once ;-)